include/ThingProperties.h

```diff
@@ -19,6 +19,9 @@ float battery_Voltage;
 float acceleration_x;
 float acceleration_y;
 float acceleration_z;
+double pressure_2;
```

Comment:
why is this called pressure_2 ?
Can we change is to pressure?

Same for temp.

Reply:
I just wanted to make a difference between the two (pressure and pressure_2), in line 44 you can see a updatePressure function, pressure is the name of the parameter (to access the parameter's pressure and temp). 
To avoid confusions, we can change the parameter name, and set the variables to pressure and temp respectively:

void updatePressure(pressure_t sensor){
    pressure = sensor.pressure;
    temp = sensor.temp;
}